src/irr/asset/CCPUMeshPacker.h

```diff
@@ -33,10 +35,17 @@ class CCPUMeshPacker final : public IMeshPacker<ICPUMeshBuffer, MDIStructType>
 };
 
 template <typename MDIStructType>
-PackedMeshBufferData CCPUMeshPacker<MDIStructType>::alloc(const ICPUMeshBuffer const* meshBuffer)
+//`Iterator` may be only an Iterator or pointer to pointer
+template <typename Iterator>
+ReservedAllocationMeshBuffers CCPUMeshPacker<MDIStructType>::alloc(const Iterator begin, const Iterator end)
 {
-	if (meshBuffer == nullptr)
-		return invalidPackedMeshBufferData;
+
+	for (auto it = begin; it != end; it++)
+	{
+		if (*it == nullptr)
+			return invalidReservedAllocationMeshBuffers;
```

Comment:
just `assert` this.

Reply:
so you want program to terminate in this case? why?

Reply:
because its a gross logic error to feed a function a list of pointers that contains a null.